.github/workflows/build_docs.yaml

```diff
@@ -106,6 +106,7 @@ jobs:
                 if [ "$RELEASE_INPUT" == "true" ]; then
                   version=$(sed -n 's/^version = "\(.*\)"/\1/p' docs/reference/conf.py)
                   sed -i "s/\$TYPESENSE_INDEX_NAME/$version/g" "$searchbox_html"
+                  sed -i 's/sitemap_url_scheme = "master\/docs\/slint\/{link}"/sitemap_url_scheme = "{version}docs\/slint\/{link}"/' docs/reference/conf.py
```

Comment:
```suggestion
                  sed -i 's/sitemap_url_scheme = "master\/docs\/slint\/{link}"/sitemap_url_scheme = "{version}/docs\/slint\/{link}"/' docs/reference/conf.py
```

Reply:
Sphinx build adds a '/' after the version when it encounters {version}
https://sphinx-sitemap.readthedocs.io/en/latest/advanced-configuration.html#customizing-the-url-scheme

Reply:
My goodness, that’s magic :)